src/components/Homepage/SecondSection.js

```diff
@@ -9,65 +9,41 @@ const SecondSection = () => {
   return (
     <div id="second-section">
       <div className="second-section-ul">
-        {/* <ul>
-          <li>
-            {" "}
-            <span className="lists">
-              <img src={logo} alt="logo" />
-            </span>{" "}
-            Track your daily activities
-          </li>
-          <li>
-            <span className="lists">
-              <img src={logo} alt="logo" />
-            </span>{" "}
-            Save important note or code snippets you're likely to reuse.
-          </li>
-          <li>
-            <span className="lists">
-              <img src={logo} alt="logo" />
-            </span>{" "}
-            Keep your dairy of your daily activities
-          </li>
-        </ul>*/}
         <div id="list1">
           <span className="lists1">
-            <img src={logo} alt="logo" />
+            <img src={ logo } alt="logo" />
```

Comment:
FYI this space is not required.
```suggestion
            <img src={logo} alt="logo" />
```

And this goes for all the changes in the component. 
You don't have to change it I just felt I should tell you